suite-native/app/package.json

```diff
@@ -1,42 +1,26 @@
 {
-    "name": "@trezor/suite-native",
+    "name": "@suite-native/app",
```

Comment:
please update workspace name in global native:* scripts https://github.com/trezor/trezor-suite/blob/chore/rn-73/package.json/#L66-L69 🙏

Reply:
Good catch

Reply:
fixed